pkg/controller/job/job_controller.go

```diff
@@ -56,6 +56,12 @@ import (
 	"k8s.io/utils/ptr"
 )
 
+const (
+	// PodFailurePolicy reason indicates a job failure condition is added due to
+	// a failed pod matching a pod failure policy rule
+	jobConditionReasonPodFailurePolicy = "PodFailurePolicy"
+)
+
```

Comment:
This was removed, please don’t add it back.

Reply:
yes, remove is done.